src/mocks.ts

```diff
@@ -29,6 +30,21 @@ export const mocks = [];`);
   return mocks as IMock[];
 }
 
+export function getMockForJSON(json: string | {}): RequestHandler {
```

Comment:
Just curious, I assume `{}` is a different way of writing `object`, right? It's not "an empty object", is it?

Reply:
Most probably I'm misusing it. I thought of it as a "plain object" but it's different actually https://blog.mariusschulz.com/2017/02/24/typescript-2-2-the-object-type#object-vs-object-vs maybe we wanna use `object`

Reply:
I am still learning TS, but I am always using object so I have no knowledge about the other way.